### ISSUE TITLE
Where are the TSE 2012 results?

### ISSUE THREAD

**Issue by clegoues** [squaresLab/genprog-code#2]
Mysteriously not linked from the old GenProg site, either.

**Comment by weimer** [comment]
>  I was hoping to dig up the actual logs, 

I agree. For GenProg 2009-based experiments, we are now talking about science from eight years ago, and we may well just have to give up on some of it.

That said, some of those archives do include logs -- but recall that the logs used to be named differently way back in the day: 

weimer@dijkstra:~/genprog/resources$ tar tf genprog-benchmarks-tse-2012.tar.gz | grep -i debug
genprog-benchmarks-tse-2012/deroff/deroff.c-reference.debug
genprog-benchmarks-tse-2012/atris/atris_comb.c-reference.debug
genprog-benchmarks-tse-2012/zune/zunebug.c-reference.debug
genprog-benchmarks-tse-2012/openldap/io.i-reference.debug
genprog-benchmarks-tse-2012/indent/indent_comb.c-reference.debug
genprog-benchmarks-tse-2012/uniq/uniq.i-reference.debug
genprog-benchmarks-tse-2012/svr-look/look.c-reference.debug
genprog-benchmarks-tse-2012/ccrypt/ccrypt_comb.c-reference.debug
genprog-benchmarks-tse-2012/gcd/gcd.c-reference.debug
genprog-benchmarks-tse-2012/flex/flex_comb.c-reference.debug
genprog-benchmarks-tse-2012/wu-ftpd/ftpd_comb.c-reference.debug
genprog-benchmarks-tse-2012/lighttpd/mod_fastcgi.i-reference.debug
genprog-benchmarks-tse-2012/nullhttpd/httpd_comb.c-reference.debug
genprog-benchmarks-tse-2012/ultrix-look/look.c-reference.debug
genprog-benchmarks-tse-2012/php/string.i-reference.debug
genprog-benchmarks-tse-2012/units/units.c-reference.debug

We'll see things like this at the bottom of one:

        fitness 15
                best so far (size delta 449)
v_gen 0
avgfit :
bcfit :


Best result written to deroff.c-0-0.01-0.06-best.c
        First Solution in 15.8366 (6 fitness evals)
        Best  Solution in 15.8366 (6 fitness evals)
Generations to solution: 1
Avg ins: 0/19 = 0
Avg del: 0/.. = 0
Avg swap: 0/.. = 0
Avg xover: 0/.. = 0
Avg mut: 0/.. = 0
Percent failed to compile: 19/6 = 3.16667

**Comment by blank-black** [comment]
> My guess, via this link from the old genprog webpage ...
> 
> TSE 2012 Benchmark Programs and workloads. The instructions assume GenProg v1.0.
> 
> ...
> 
> is:
> 
> http://dijkstra.cs.virginia.edu/genprog/resources/genprog-benchmarks-tse-2012.tar.gz
> 
> http://dijkstra.cs.virginia.edu/genprog/resources/genprog-tse2012-workload.tar.gz
> 
> Similarly, via
> 
> GenProg TSE 2012 virtual machine image and instructions for a WU-FTP format string vulnerability repair demo
> 
> http://dijkstra.cs.virginia.edu/genprog/resources/genprog-tse2012-wuftpd.vdi
> 
> http://dijkstra.cs.virginia.edu/genprog/resources/genprog-tse2012-wuftpd.txt

Hi, the links here are expired.
Do you have the new link?

Thanks!
